.github/workflows/test.yml

```diff
@@ -1,283 +1,68 @@
-name: fastdds-statistics-backend-tests
+# Tests to run with workflow-dispatch, pull-request and push events
+name: test
 
 on:
-
   workflow_dispatch:
     inputs:
 
-      dependencies_artifact_postfix:
-        description: 'Postfix name to add to artifact name to download dependencies. This is use to download a specific artifact version from eProsima-CI.'
+      fastcdr_branch:
```

Comment:
Is it necessary to have the repository branches as inputs in the workflow?

Reply:
You are right